### DISCUSSION TITLE
Super table support for CSV

### DISCUSSION THREAD

**Discussion by cryptomothy** [engram-design/FeedMe#888] [Ideas]
### Description

I'm trying to import a CSV file (~ 150 rows), on which some columns have multiple values to import inside a Super table. 
I have set up the dataDelimiter to "|" (the pipe) inside a feed-me.php file in my config folder. 

```
<?php

return [
	'*' => [
		'pluginName' => 'Feed Me',
		'dataDelimiter' => '|',
	]
];

```

When i run the feed, only one row is created inside the table, containing an array of data, instead of breaking every entry in that said array into multiple rows.

The data as it is inside the CSV:

`actuaire|actuaire (fonction publique québécoise)|actuaire associé/actuaire associée|actuaire en assurance-accidents, risques divers`

And here is the array after import:

`["actuaire","actuaire (fonction publique qu\u00e9b\u00e9coise)","actuaire associ\u00e9\/actuaire associ\u00e9e","actuaire en assurance-accidents, risques divers"]`

![image](https://user-images.githubusercontent.com/14184116/51490384-1f9ad580-1d79-11e9-8b82-df323f399de9.png)

(way more entries in some cases)

And i have tried multiple different delimiters...

### Steps to reproduce

1. Create a CSV file with multiple entries,
2. Some of them need to have a column which contains multiple values to be imported inside a Super table field, separated with a pipe "|",
3. Create a feed-me.php file under your /config directory to configure the dataDelimiter,
4. Run the feed.

### Additional info

- Plugin version: Feed Me 3.0.2.1
- Craft version: Craft CMS 3.0.37

**Comment by engram-design** [comment]
Currently, I'm afraid Super Table and Matrix aren't fully supported with delimiters for CSV importing. But, its a requested feature, it'll just require a bit of re-architecting  https://github.com/verbb/feed-me/issues/368, https://github.com/verbb/feed-me/issues/314
